backend/tests/unit/simulation/alarms.txt

```diff
@@ -0,0 +1,34 @@
+Scenario: alarm services behaving properly
+Given: a pc_ac alarm service object 
+When: parameter mode is not equal to pc_ac
+Then: alarms are not updated for parameters
+
+Scenario: alarm services behaving properly
+Given: a pc_ac alarm service object 
+When: parameter ventilating mode is false
+Then: alarms are not updated for parameters
```

Comment:
My understanding is that the alarms are actually updated - all active alarms are deactivated.

Reply:
Yes! will update it.